packages/components/src/List/Toolbar/Toolbar.scss

```diff
@@ -27,11 +29,63 @@ $tc-list-toolbar-background-color: $concrete !default;
 		}
 	}
 
-	:global(.navbar-nav) {
-		a[role="menuitem"],
+	:global(.nav) {
+		a[role='menuitem'],
 		a:global(.dropdown-toggle) {
 			color: $black;
 			text-transform: capitalize;
+			font-weight: normal;
+		}
+	}
+
+	:global(.tc-actionbar-container) {
+		background-color: $tc-list-toolbar-background-color;
+		padding-right: 0;
+		height: $tc-list-toolbar-height;
+
+		:global(.navbar-left),
+		:global(.navbar-right) {
+			> * {
+				margin-right: $padding-normal;
+			}
+		}
+	}
+
+	:global(.list-action-bar) {
+		:global(.navbar-right) {
+			& > ul {
+				list-style: none;
+				display: flex;
+				align-items: center;
+				margin: 0 $padding-small 0 0;
```

Comment:
Setting a part of the space in different places doesn't seem ok.
Can you remove this margin-right please, to set it to 0.

To manage the space on the right part of the bar : 
* for each element, don't define a space on the right. Make sure it is set to 0 (so it is exactly on the right, the $padding-larger that you have to set on the whole bar will define the the right space.
* for each element, make sure that the left space is ok. Then add a margin of the same space on the left, so it defines the right space of the previous element.
* you will have issues with separator, perhaps you can reverse the the logic. For each element, add a separator (:before instead of :after), and remove it for the first element.